_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+
+### Getting prepared
+
+Just as last year, our Scrum Masters did an outstanding job organizing and ensuring a smooth flow of the two-day hackathon.
+They divided up teams that usually don't work with each other on a regular basis, they agreed upon final topics, booked meeting rooms for the teams and most importantly ordered delicious snacks, drinks and food {% emoji wink %}.
+
+A day before the hackathon, our colleagues from Jena and Barcelona arrived in Hamburg to join the hacking fun at our headquarter.
+
+### Let's get started!
+
+The first day started off with a surprisingly blue sky and a vitalizing breakfast to get all developers fueled and ready to hack away.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Ready to hack away! Starting off with a breakfast! <a href="https://twitter.com/hashtag/ephack17?src=hash">#ephack17</a> <a href="https://t.co/Gf552CgNc9">pic.twitter.com/Gf552CgNc9</a></p>&mdash; ePages Developers (@epagesdevs) <a href="https://twitter.com/epagesdevs/status/844451431872647168">March 22, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After everyone's stomach was filled, the developers were ready to get started in one of the 13 pre-assigned teams, which all addressed different topic areas.
+The whole day was determined by concentration, creativity and of course lots of fun and laughter.
+Some teams got so hooked up and passionate about their project that they even worked until the wee hours of the morning and continued early in the morning the next day.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/HACKATHON?src=hash">#HACKATHON</a> <a href="https://twitter.com/epagesdevs">@epagesdevs</a> at 3.30 AM: Everyone left for beer.. BUT <a href="https://twitter.com/hashtag/smartshop?src=hash">#smartshop</a> hackers are <a href="https://twitter.com/hashtag/unstoppable?src=hash">#unstoppable</a> <a href="https://twitter.com/hashtag/ephack17?src=hash">#ephack17</a> <a href="https://twitter.com/oozpy">@oozpy</a> <a href="https://t.co/xui2qukGNE">pic.twitter.com/xui2qukGNE</a></p>&mdash; Benjamin Nothdurft (@dataduke) <a href="https://twitter.com/dataduke/status/844816040454307840">March 23, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+No wonder then, that the results the next day in the evening were brilliant:
+
+### Final presentations
+
+During the two days of the hackathon the teams mainly focused on pushing forward our new architecture Beyond, enhancing our current shop software ePages 6 and improving internal processes.
+The audience was quite impressed by the creative outcomes of every single team.
+
+For further developing our newest backend architecture some teams dedicated the two days on simplifying the interaction with the Beyond API with the help of the query language [GraphQl](http://graphql.org/learn/).
```

Comment:
To try out some new ideas for BEYOND, some teams dedicated the two days to simplify API interaction by using the query language...